propn-ner-comparator/parse.py

```diff
@@ -0,0 +1,41 @@
+import typer
+from conllu import parse
+from pathlib import Path
+from sklearn.metrics import confusion_matrix
+
+def parse_compare(conllu_file: Path, iob_file: Path):
+    
+    conllu_sentences = parse(conllu_file.read_text())
+    iob_sentences = parse(iob_file.read_text())
+
+    n_tokens = 0
+    n_misaligned = 0
+
+    propn_arr = []
+    tagged_arr = []
+
+    for conllu_s, iob_s in zip(conllu_sentences, iob_sentences):
+        for conllu_tok, iob_tok in zip(conllu_s, iob_s):
+            n_tokens += 1
+            if str(conllu_tok) != str(iob_tok):
+                n_misaligned += 1
+            else:
```

Comment:
@mayhewsw the way this is structured makes it so that the coverage/overlap is only computed for tokens which seem to be aligned. Happy to put it together in any other way that would make sense.

Reply:
This makes sense!